Updated supported platforms with limited support details. Signed-off-by: Sudharshan Kaushik Kannan <skk@progress.com> (#4016)
chef/chef-web-docs: content/platforms.md

Many of these platforms are forks, clones, or otherwise derivative of platforms that Chef commercially supports. Continued functionality for these platforms is likely, but not guaranteed. Unsupported platforms may have missing or non-operative functionality. As always, we welcome community contributions from anyone looking to expand community support for platforms in Chef products.

> ### Derived platforms support
> 
> Platforms that are in essence the same fundamental packages as of a counterpart platform that falls under Commercial Support are classified as derived platforms. This will mean that only the binaries of the parent platform will be tested and fully supported. Chef does not explicitly test derived platforms as part of the development and release process. The derived platform will only be supported until any point of divergence it may have from it's parent. Continued functionality for these platforms is likely, but not guaranteed. 

Reviewer: ```suggestion
### Support for Derived Platforms

Derived platforms are platforms that are fundamentally the same as a commercially supported platform. Chef does not explicitly test derived platforms as part of our development and release process, only commercially supported platforms are tested. Derived platforms are only supported until that platform diverges from a commercially supported parent platform. Continued functionality for derived platforms is likely, but not guaranteed.
```

</tbody>
</table>

#### Derived platforms

> The following table lists the platforms and versions classified under Derived Platforms for Chef Infra Client:

Reviewer: ```suggestion
The following table lists supported derived platforms and versions for Chef Infra Client.

See our policy on [support for derived platforms](#support-for-derived-platforms) for more information.
```

</tbody>
</table>

#### Derived platforms

> The following table lists the platforms and versions classified under Derived Platforms for Chef Workstation:

Reviewer: ```suggestion
The following table lists supported derived platforms and versions for Chef Workstation.

See our policy on [support for derived platforms](#support-for-derived-platforms) for more information.
```

</tbody>
</table>

#### Derived platforms

> The following table lists the platforms and versions classified under Derived Platforms for Chef Inspec:

Reviewer: ```suggestion
The following table lists supported derived platforms and versions for Chef InSpec.

See our policy on [support for derived platforms](#support-for-derived-platforms) for more information.
```

</tbody>
</table>

#### Derived platforms

The following table lists supported derived platforms and versions for Chef InSpec.

See our policy on [support for derived platforms](#support-for-derived-platforms) for more information.

<table>
<thead>
<tr class="header">
<th>Platform</th>
<th>Architecture</th>
<th>Version</th>
<th>Parent platform</th>
</tr>
</thead>
<tbody>
<tr>
<td>AlmaLinux</td>
<td><code>x86_64</code>, <code>aarch64</code></td>
<td><code>8.x</code></td>
<td>CentOS</td>
</tr>
<tr>
<td>Rocky Linux</td>
<td><code>x86_64</code>, <code>aarch64</code></td>
<td><code>8.x</code></td>
<td>CentOS</td>
</tr>
</tbody>
</table>

> 

Reviewer: ```suggestion
```

</tbody>
</table>

#### Derived platforms

> The following table lists the platforms and versions classified under Derived Platforms for Chef Infra Server:

Reviewer: ```suggestion
The following table lists supported derived platforms and versions for Chef Infra Server.

See our policy on [support for derived platforms](#support-for-derived-platforms) for more information.
```

</tbody>
</table>

#### Derived platforms

> The following table lists the platforms and versions classified under Derived Platforms for Chef Manage:

Reviewer: ```suggestion
The following table lists supported derived platforms and versions for Chef Manage.

See our policy on [support for derived platforms](#support-for-derived-platforms) for more information.
```
